packages/core/package.json

```
@@ -0,0 +1,224 @@
+    },
+    "mariadb": {
+      "optional": true
+    },
+    "sqlite3": {
+      "optional": true
+    },
+    "tedious": {
+      "optional": true
+    }
+  },
+  "keywords": [
+    "mysql",
+    "mariadb",
+    "sqlite",
+    "postgresql",
+    "postgres",
+    "pg",
+    "mssql",
+    "db2",
+    "ibm_db",
+    "sql",
+    "sqlserver",
+    "snowflake",
+    "orm",
+    "nodejs",
+    "object relational mapper",
+    "database",
+    "db"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "tag": "alpha"
+  },
+  "scripts": {
+    "----------------------------------------- static analysis -----------------------------------------": "",
+    "test-typings": "tsc --noEmit --emitDeclarationOnly false && tsc -b test/tsconfig.json",
+    "----------------------------------------- tests ---------------------------------------------------": "",
+    "mocha": "mocha -r ./test/registerEsbuild",
+    "test-unit": "yarn mocha \"test/unit/**/*.test.[tj]s\"",
+    "test-integration": "yarn mocha \"test/integration/**/*.test.[tj]s\"",
+    "test-smoke": "yarn mocha \"test/smoke/**/*.test.[tj]s\" --timeout 600000",
+    "teaser": "ts-node test/teaser.ts",
+    "test": "yarn prepare && yarn test-typings && yarn teaser && yarn test-unit && yarn test-integration",
+    "----------------------------------------- coverage ------------------------------------------------": "",
+    "cover": "rimraf coverage && yarn teaser && yarn cover-integration && yarn cover-unit && yarn merge-coverage",
+    "cover-integration": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/integration/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/integration.info')\"",
+    "cover-unit": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/unit/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
+    "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
+    "----------------------------------------- unit tests ---------------------------------------------": "",
+    "test-unit-mariadb": "cross-env DIALECT=mariadb yarn test-unit",
+    "test-unit-mysql": "cross-env DIALECT=mysql yarn test-unit",
+    "test-unit-postgres": "cross-env DIALECT=postgres yarn test-unit",
+    "test-unit-sqlite": "cross-env DIALECT=sqlite yarn test-unit",
+    "test-unit-mssql": "cross-env DIALECT=mssql yarn test-unit",
+    "test-unit-db2": "cross-env DIALECT=db2 yarn test-unit",
+    "test-unit-ibmi": "cross-env DIALECT=ibmi yarn test-unit",
+    "test-unit-snowflake": "cross-env DIALECT=snowflake yarn test-unit",
+    "test-unit-all": "yarn test-unit-mariadb && yarn test-unit-mysql && yarn test-unit-postgres && yarn test-unit-mssql && yarn test-unit-sqlite && yarn test-unit-snowflake && yarn test-unit-db2 && yarn test-unit-ibmi",
+    "----------------------------------------- integration tests ---------------------------------------------": "",
```

```suggestion
    "---------------------------------------- integration tests --------------------------------------------": "",
```
Or however long we need it to be to the same size as the others

---

Done